### ISSUE TITLE
Header - Add landmark region of banner

### ISSUE THREAD

**Issue by BogdanDogaru** [bbc/simorgh#345]
**Is your feature request related to a problem? Please describe.**
Header component does not have a landmark region of banner. It should also have announced text of 'BBC News'. 

**Describe the solution you'd like**
We should set the Header element to have a landmark region of banner and aria label of BBC News.
We should also test this with a screenreader. Instructions here: https://bbc-news.github.io/accessibility-news-and-you/accessibility-and-testing-with-assistive-technology

**Additional context**
Example of landmark usage: https://www.w3.org/TR/wai-aria-practices/examples/landmarks/banner.html 
Example of `aria-label` usage: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute

**Comment by PaulChinkwende** [comment]
Further information on the closure of this issue can be found in the conversation of https://github.com/bbc/simorgh/pull/383.
